plugins/generator-1.20.1/forge-1.20.1/block.definition.yaml

```diff
@@ -401,6 +401,16 @@ tags:
     condition: "${data.destroyTool == 'shovel'}"
   - tag: BLOCKS:minecraft:mineable/hoe
     condition: "${data.destroyTool == 'hoe'}"
+
+  - tag: BLOCKS:minecraft:needs_stone_tool
+    condition: "${data.requiresCorrectTool && data.breakHarvestLevel == 1}"
+  - tag: BLOCKS:minecraft:needs_iron_tool
+    condition: "${data.requiresCorrectTool && data.breakHarvestLevel == 2}"
+  - tag: BLOCKS:minecraft:needs_diamond_tool
+    condition: "${data.requiresCorrectTool && data.breakHarvestLevel == 3}"
+  - tag: BLOCKS:minecraft:needs_netherite_tool

```

Comment:
Vanilla Minecraft doesn't have a tag for netherite tools. It's instead added by Forge/NeoForge, in both cases with the `forge` namespace (will move to the `c` namespace in 1.21)

Reply:
Good catch, will fix